src/test/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpointUnitTest.java

```diff
@@ -328,4 +402,16 @@ public void shouldRejectFulfilByPostWithIncorrectRequestBody() throws Exception
     mockMvc.perform(postJson(url, json)).andExpect(status().isBadRequest());
     verify(caseService, never()).fulfilmentRequestByPost(any(PostalFulfilmentRequestDTO.class));
   }
+
+  private void submitInvalidNewCaseRequest(CaseRequestDTO newCaseRequest)
+      throws Exception, JsonProcessingException {
```

Comment:
JsonProcessingException not required if you are going to throw Exception as well, so it can either be removed or the more specific exceptions added to the throws clause

Reply:
Removed